configure.ac

```diff
@@ -123,6 +123,7 @@ OPTION_DEFAULT_ENABLE([ovis_ctrl], [ENABLE_OVIS_CTRL])
 OPTION_DEFAULT_ENABLE([ovis_util], [ENABLE_OVIS_UTIL])
 OPTION_DEFAULT_ENABLE([ovis_auth], [ENABLE_OVIS_AUTH])
 OPTION_DEFAULT_ENABLE([zap], [ENABLE_ZAP])
+OPTION_DEFAULT_DISABLE([rdc], [ENABLE_RDC])
```

Comment:
I believe you alluded to this in your PR comment about "fixing" configure.ac. We really shouldn't be using OPTION_DEFAULT_DISABLE for this. We instead need a proper library test for the rdc library. This plugin would just be enabled when the required libraries are included.

Reply:
agree. just don't have a valid environment yet to develop that.